functions/src/notifications/publishNotifications.ts

```diff
@@ -88,62 +87,54 @@ export const publishNotifications = functions.firestore
     console.log(`topic type: ${topic.type}`)
 
     const handleNotifications = async (
-      topic: BillNotification | OrgNotification
+      topic: BillHistoryUpdateNotification | TestimonySubmissionNotification
     ) => {
       const notificationFields = createNotificationFields(topic)
 
-      console.log(JSON.stringify(notificationFields))
-
       const topicNameSnapshot = await db
         .collectionGroup("activeTopicSubscriptions")
-        .where("topicName", "==", notificationFields.topicName)
+        .where(
+          "topicName",
+          "in",
+          topic.type === "bill"
+            ? [`bill-${topic.billCourt}-${topic.billId}`]
+            : [
+                `testimony-${topic.orgId}`,
+                `bill-${topic.billCourt}-${topic.billId}`
+              ]
+        )
         .get()
 
-      // Send a testimony notification to all users subscribed to the Bill
-      let billSnapshot
-      if (notificationFields.notification.type !== "bill") {
-        billSnapshot = await db
-          .collectionGroup("activeTopicSubscriptions")
-          .where(
-            "topicName",
-            "==",
-            `bill-${notificationFields.notification.court}-${notificationFields.notification.id}`
-          )
-          .get()
-      }
-
-      const uniqueDocs = new Map()
-
-      // Add documents from topicNameSnapshot to the Map
-      topicNameSnapshot.docs.forEach(doc => {
-        uniqueDocs.set(doc.data().uid, doc.data())
+      const users: { [uid: string]: any } = {}
+
+      // Iterate through the topicNameSnapshots and set the notifications
+      topicNameSnapshot.forEach(subscription => {
+        const { uid, type } = subscription.data()
+        if (topic.type === "testimony" && topic.orgId !== uid) {
```

Comment:
nit: While we're updating terms here, it might be good to cut over `orgId` and `orgLookup` to something broader (e.g. `userId` and `userLookup`, possibly with an extra `isOrg` flag if it feels necessary) to keep our terms consistent.